database/src/migrations/20241012021000_survey_jobs.ts

```diff
@@ -0,0 +1,50 @@
+import { Knex } from 'knex';
+
+/**
+ *  add crew member and crew lead
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+    
+    SET SEARCH_PATH=biohub
+    
+    ----------------------------------------------------------------------------------------
+    -- Adding survey participant roles
+    ----------------------------------------------------------------------------------------
+        INSERT INTO biohub.survey_job (name, record_effective_date, description)
+        VALUES
+          ('Crew lead', NOW(), 'A participant of a survey in a crew lead role.'),
+          ('Crew member', NOW(), 'A participant of a survey in a crew member role.');
+
+    ----------------------------------------------------------------------------------------
+    -- Reassigning sims biologist roles to crew member
+    ----------------------------------------------------------------------------------------
+      
+        UPDATE biohub.survey_participation sp
```

Comment:
The migration is failing here because you can't give your table an alias in this part of the update. You can drop the table references, and you since every table touched is in the biohub schema that you set for the search path, you can drop the schema reference:

```UPDATE survey_participation
        SET survey_job_id = (
            SELECT survey_job_id 
            FROM survey_job
            WHERE name = 'Crew member'
        )
        WHERE survey_job_id = (
            SELECT survey_job_id 
            FROM survey_job
            WHERE name = 'Biologist'
        );
```